pkg/timestamp/microts.go

```diff
@@ -76,10 +77,7 @@ func (ts MicroTS) Protobuf() *timestamp.Timestamp {
 
 // GogoProtobuf converts this microtimestamp to a (Gogo) protobuf representation.
 func (ts MicroTS) GogoProtobuf() *types.Timestamp {
```

Comment:
As I remember, our goal with the compatibility layer is to remove `gogo` dependency from our code:
```
	"github.com/gogo/protobuf/types"
```

And keep it isolated only in`protocompat` layer to have a single place to change when we switch to another generator. This package still uses `gogo/protobuf/types`.

Is this addressed in some follow-up PR?
Or should we adjust this to add compatibility layer where `GogoProtobuf()` is called.
Theoretically, we should remove `GogoProtobuf()`. But also need to check if that "interface" is used in scanner.

Reply:
I plan a follow-up PR where the remaining usage of "github.com/gogo/protobuf" sublibraries out of the proto* packages is removed. This one is not out yet, though.